deploy/shakenfist_ci/tests/test_floating_ips.py

```diff
@@ -64,14 +64,22 @@ def test_simple(self):
                 console = base.LoggingSocket(
                     inst['node'], inst['console_port'])
                 out = console.execute('cat /var/www/html/index.html')
-                if out.find('Floating IPs work!') != -1:
+                if out.find('Floating IPs work!') == -1:
+                    print('Floating IPs test attempt failed, userdata has not run')
+                else:
                     r = requests.request(
                         'GET', 'http://%s/' % ifaces[0]['floating'])
-                    if r.status_code == 200 and r.text.find('Floating IPs work!') != -1:
+                    print('Floating IPs test attempt received HTTP status %s' %
+                          r.status_code)
+                    if r.status_code == 200 and r.text.find('Floating IPs work!') == -1:
+                        print(
+                            'Floating IPs test attempt failed, incorrect HTTP result')
+                    else:
+                        print('Floating IPs test passed')
```

Comment:
Should this pass when the status code is not 200?

Reply:
No. Good catch.